manifests/bootkube/config/config-overrides.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: kubecontrolplane.config.openshift.io/v1
+kind: KubeControllerManagerConfig
+serviceServingCert:
+  certFile: service-signer.crt
+extendedArguments:
+  cloud-config:
+  - "/etc/kubernetes/cloud/config"
+  root-ca-file:
+  - "/etc/kubernetes/secrets/root-ca.crt"
+  service-account-private-key-file:
+  - "/etc/kubernetes/secrets/service-account.key"
+  cluster-signing-cert-file:
+  - "/etc/kubernetes/secrets/kube-ca.crt"
```

Comment:
in some followup, I think we should split secrets like we did in the apiserver

Reply:
rule of thumb right now: follow kube-core to make replacing it easy. After we have done that we can make everything pretty.